### PR TITLE
Use RSpec and VCR in place of minitest/webmock

### PR DESCRIPTION
**Pull request by erithmetic**

### CODE REVIEW

File: spec/carbon_spec.rb [hunk 1]
```diff
@@ -45,59 +47,69 @@ def  nil_model; nil          end
...
   end
 
-  describe :query do
-    describe '(one at a time)' do
+  describe '.query' do
+    context 'serial' do
       it "calculates flight impact" do
-        result = Carbon.query('Flight', :origin_airport => 'LAX', :destination_airport => 'SFO', :segments_per_trip => 1, :trips => 1)
-        result.decisions.carbon.object.value.must_be_close_to 200, 50
+        VCR.use_cassette 'LAX->SFO flight', :record => :once do
```
Comment (seamusabshere) [inline spec/carbon_spec.rb:53]: hah fascinating

File: spec/cassettes/Flight.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (seamusabshere) [inline spec/cassettes/Flight.yml:1]: how are these cassettes created?

File: spec/cassettes/flight_with_key_2.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: http://impact.brighterplanet.com/flights.json
+    body:
+      encoding: US-ASCII
```
Comment (seamusabshere) [inline spec/cassettes/flight_with_key_2.yml:7]: holy crap are we really returning US-ASCII not utf-8?

File: spec/cassettes/Flight.yml (hunk 2 above)
Comment (erithmetic) [inline spec/cassettes/Flight.yml:1]: when an http request is made, VCR checks if a cassette exists, if not and the record option is :all, :once, or :new_episodes, it saves the response in an appropriately named file. If the record option is :new_episodes, it puts any "new" http requests (i.e. requests that don't match previous method/uri's recorded) then it adds them to the cassette. If option is :once, any new requests raise an error.

**Comment by seamusabshere** [comment]
1. please note changes in changelog
2. please make plain old `rake` run all the specs

**Comment by seamusabshere** [comment]
tk you @dkastner, an elegant pull request

**Pull request merged**